webpack.config.js

```diff
@@ -1,52 +1,90 @@
-const hash = require('child_process')
-  .execSync('git rev-parse --short HEAD')
-  .toString().replace(/(\r\n|\n|\r)/gm, '')
+const path = require("path");
 
-module.exports = {
-  entry: './src/index.js',
-  watch: true,
-  devtool: process.env.DEBUG ? 'eval-source-map' : undefined,
-  module: {
-    rules: [
-      {
-        test: /\.(js|jsx)$/,
-        exclude: /node_modules/,
-        use: ['babel-loader']
-      },
-      {
-         test: /\.(woff|woff2|eot|ttf|otf)$/,
-         use: [ 'file-loader']
-       }
-    ]
-  },
-  externals: {
-    react: {
-      root: 'React',
-      commonjs2: 'react',
-      commonjs: 'react',
-      amd: 'react'
-    },
-    'react-dom': {
-      root: 'ReactDOM',
-      commonjs2: 'react-dom',
-      commonjs: 'react-dom',
-      amd: 'react-dom'
-    }
-  },
-  resolve: {
-    extensions: ['*', '.js', '.jsx']
-  },
-  output: {
-    path: __dirname + '/../kong-portal-templates/workspaces/default/themes/base/assets/js',
-    publicPath: '/',
-    filename: `swagger-ui-kong-theme-${hash}.${process.env.DEBUG ? 'debug' : 'min'}.js`,
-    library: 'SwaggerUIKongTheme',
-    libraryTarget: 'umd'
+const hash = require("child_process")
+  .execSync("git rev-parse --short HEAD")
+  .toString()
+  .replace(/(\r\n|\n|\r)/gm, "");
+
+const webpack = require("webpack");
+
+const outputs = {
+  development: {
+    path: path.join(__dirname, "dist"),
+    publicPath: "/",
+    filename: `bundle.js`,
+    library: "SwaggerUIKongTheme",
+    libraryTarget: "umd",
   },
-  devServer: {
-    contentBase: './dist'
+  production: {
+    path: path.join(
+      __dirname,
+      "/../kong-portal-templates/workspaces/default/themes/base/assets/js"
+    ),
+    publicPath: "/",
+    filename: `swagger-ui-kong-theme-${hash}.${
+      process.env.DEBUG ? "debug" : "min"
+    }.js`,
+    library: "SwaggerUIKongTheme",
+    libraryTarget: "umd",
   },
-  optimization: process.env.DEBUG ? {
-    minimize: false
-  } : undefined
-}
+};
+
+module.exports = (env, argv) => {
+  return {
+    entry: "./src/index.js",
+    watch: true,
+    devtool: process.env.DEBUG ? "eval-source-map" : undefined,
```

Comment:
Don't use `process.env` when you're using webpack config function export and you have `env` provided as an argument in line 32